src/Symfony/Component/Form/Tests/Extension/Validator/ViolationMapper/ViolationMapperTest.php

```diff
@@ -1738,4 +1738,27 @@ public function testMessageWithLabelFormat2()
             $this->assertSame('Message Translated 2nd Custom Label', $error->getMessage());
         }
     }
+
+    public function testTranslatorNotCalledWithoutLabel()
+    {
+        $renderer = $this->getMockBuilder(FormRenderer::class)
+            ->setMethods(null)
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+        $translator = $this->getMockBuilder(TranslatorInterface::class)->getMock();
+        $translator->expects($this->any())->method('trans')->willReturnCallback(function () {
```

Comment:
```php
$translator = $this->createMock(TranslatorInterface::class);
$translator->expects($this->never())->method('trans');
```

What about this instead?

Reply:
Sure! I'm not supra familiar with PHPUnit mocks, your version is quite better.